### DISCUSSION TITLE
Lagginess when using Move shortcut

### DISCUSSION THREAD

**Discussion by greenerycomplexity** [rxhanson/RectanglePro-Community#504] [Q&A] [answered]
Long-time Rectangle user here, recently moved to Rectangle Pro and been loving all the new shortcuts, especially _Move_ and _Resize_ window function, under the Window Throw section.

However, I've been noticing that the Move can be weirdly laggy/jittery sometimes, felt like I was looking at my windows moving in sub-30 FPS. This has never been a problem where I normally go to the top of a window and drag it using my mouse, so the issue is something to do with Rectangle Pro.

Would be great if a fix could be sent out to patch this. Appreciate it lots!

**Comment by rxhanson** [comment, marked as answer]
Unfortunately, this is a limitation of using what's available as a 3rd party on macOS. If someday a new API opened up that gave access to what's available internally in macOS, then I would switch over.

(Thanks for making the jump to Rectangle Pro!)
